src/modules/UI/components/PinInput/PinInput.ui.js

```diff
@@ -1,21 +1,34 @@
 // @flow
 
 import React, { Component } from 'react'
-import { TextInput, View } from 'react-native'
+import { StyleSheet } from 'react-native'
+import { TextField } from 'react-native-material-textfield'
```

Comment:
Should you be using Allen's `formField` component here?
`src/components/FormField.js`